rl_algorithms/common/buffer/wrapper.py

```diff
@@ -119,11 +120,23 @@ def sample(self, beta: float = 0.4) -> Tuple[torch.Tensor, ...]:
         weights = np.array(weights_)
         eps_d = np.array(eps_d)
 
-        weights = weights.reshape(-1, 1)
-
-        states, actions, rewards, next_states, dones = self.buffer.sample(indices)
-
-        return states, actions, rewards, next_states, dones, weights, indices, eps_d
+        if isinstance(self.buffer, RecurrentReplayBuffer):
```

Comment:
should modify return to be changed one tuple.

Reply:
resolved in commit 0949a63.